src/helpers/emailTemplate.js

```diff
@@ -0,0 +1,55 @@
+const template = ({ url, token }) => `
+        <html>
+        <head>
+          <style>
+            .title {
+              font-family: Lancelot;
+              font-size: 30px;
+              color: #f9a826;
+              margin-bottom: 30px;
+            }
+            .container {
+              font-size: 18px;
+              font-family: Lato;
+              padding: 30px;
+              border: #000 1px solid;
+              color: #728191;
+              border-radius: 20px;
+              box-shadow: 0px 0px 10px gray;
+            }
+            .btn {
+              background-color: #f9a826;
+              text-align: center;
+              padding: 10px;
+              border-radius: 10px;
+              text-decoration: none;
+              color: #000;
```

Comment:
Can you rename this file, to something better. Name it more in line with what the actual file is doing

Reply:
I named it `linkToResetPasswordTemplate`

Reply:
I named it `linkToResetPasswordTemplate.js`